spec/services/census_authorization_handler_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   before do
-    stub_request(:get, /census\.url/)
+    stub_request(:get, /example.net\/census\/url/)
```

Comment:
I've changed this to use a more realistic domain than `80`, check also the change in `secrets.yml`